include/gsl/gsl_byte

```diff
@@ -52,69 +48,69 @@ enum class byte : unsigned char
 };
 
 template <class IntegerType, class = std::enable_if_t<std::is_integral<IntegerType>::value>>
-inline constexpr byte& operator<<=(byte& b, IntegerType shift) noexcept
+constexpr byte& operator<<=(byte& b, IntegerType shift) noexcept
 {
     return b = byte(static_cast<unsigned char>(b) << shift);
 }
 
 template <class IntegerType, class = std::enable_if_t<std::is_integral<IntegerType>::value>>
-inline constexpr byte operator<<(byte b, IntegerType shift) noexcept
+constexpr byte operator<<(byte b, IntegerType shift) noexcept
```

Comment:
If you disagree with the argument presented [above](https://github.com/Microsoft/GSL/pull/446#discussion_r99503826), I'll revert these.